doc/ref/copyright.xml

```diff
@@ -66,28 +66,20 @@ make.
 In addition to the general copyright for &GAP; set forth above,
 the following terms apply to the versions of &GAP; for Windows.
 <P/>
-The executable of &GAP; for Windows that we distribute was compiled with
+&GAP; for Windows is packaged with several packages from
+<Package>Cygwin</Package> (<URL>https://cygwin.com</URL>).
+Files from <Package>Cygwin</Package> are provided under their respective
+licenses (included).
```

Comment:
What does "included" mean here?

Reply:
I don't know, we can ask @ChrisJefferson – this is part of the text I took from his suggestion https://github.com/gap-system/gap/issues/4543#issuecomment-856092232

Reply:
@ChrisJefferson indeed, looks to me like we include all licenses for all types represented there (?) - I am not sure we do.

Reply:
By included I meant (included in the release) -- as we now package a full cygwin distro, it includes all the license files (as I don't go out of my way to remove them).

Reply:
Thanks Chris, I've changed "...licenses (included)" to "...licenses, which are included in the installation." Hope that's right.